fitsio/fitslib.py

```diff
@@ -4244,7 +4271,11 @@ def add_record(self, record_in):
 
             If the input is a card string, convert is implied True
         """
-        record = FITSRecord(record_in, convert=self.convert)
+        if isinstance(record_in,dict) and 'name' in record_in and 'value' in record_in:
+            record = {}
+            record.update(record_in)
+        else:
+            record = FITSRecord(record_in)
```

Comment:
A comment on what case this `if...else` is catching would be helpful.

Reply:
its just an optimization

Reply:
I’m not following but that’s ok.